Robot2018/src/org/usfirst/frc/team199/Robot2018/RobotMap.java

```diff
@@ -1,4 +1,23 @@
-package org.usfirst.frc.team199.Robot2018;
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2017-2018 FIRST. All Rights Reserved.                        */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package org.usfirst.frc.team199.robot;
```

Comment:
should be Robot2018 at the end. Kevin went through and changed them all in the old one, so if you could change your version to match that would be good